var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -300,6 +300,19 @@ class QuantumEspresso(CMakePackage, Package):
 
     conflicts("@6.5:", when="+environ", msg="6.4.x is the latest QE series supported by Environ")
 
+    conflicts(
+        "@:7.3.0",
+        when="build_system=generic %oneapi",
+        msg="Support for ifx has been added to configure in release 7.3.1",
+    )
+    # TODO: When adding the next version make sure https://github.com/libmbd/libmbd/pull/60 is
```

Comment:
https://github.com/libmbd/libmbd/pull/60 has been merged in the QE develop and will be shipped with the next release.

Reply:
OK, I changed the comment.